ARCHITECTURE.md

```diff
@@ -6,15 +6,18 @@ This document describes the high level architecture of the av-pairtree program.
 
 The av-pairtree program is an application that watches a "drop box" for new or updated CSV files with a certain structure. When a CSV file with A/V files has been put into the drop box, the av-pairtree program picks them up and processes their audio and video files.
 
-The processing of video files (which are MP4s by default) involves putting them into a [Pairtree](https://tools.ietf.org/html/draft-kunze-pairtree-01) structure that's accessible to a media server. The processing of audio files (which are WAVs by default) involves converting the WAV files into MP4 files and, then, putting them into the Pairtree structure.
+The processing of video files (which are MP4s by default) involves putting them into a [Pairtree](https://tools.ietf.org/html/draft-kunze-pairtree-01) structure that's accessible to a media server. The processing of audio files (which are WAVs by default) involves two conversions:
 
-After all the A/V files in a CSV file have been processed, the input CSV is updated to include the resources' new access URLs (i.e. the URLs of the media files as served by the media server) and written back out to the file system.
+  * to MP4 format, the results of which are put into the Pairtree structure; and
+  * to binary [audiowaveform](https://github.com/bbc/audiowaveform) format, the results of which are deposited to AWS S3.
+
+After all the A/V files in a CSV file have been processed, the input CSV is updated to include the resources' new access URLs (i.e. the URLs of the media files as served by the media server) and audiowaveform URLs, then written back out to the file system.
 
 ![Overview diagram for av-pairtree's components](docs/images/av_pairtree_overview.svg)
```

Comment:
I realize now that I also need to update the architecture diagram -- could that task be a separate ticket?

Reply:
Yes, just create the new ticket for it.

Reply:
https://jira.library.ucla.edu/browse/SERV-167